### ISSUE TITLE
Collection of player command

### ISSUE THREAD

**Issue by Vonfry** [SpringHan/netease-cloud-music.el#3]
Could we list some useful player commands for `netease-cloud-music-player-command` in readme or here? By the way, would it 
 be defined by `defcustom` instead `defvar`.

Anyone can put your player config here. unsubscribe from notifications if you don't wish to be distrubed.

***mpv***
`'("mpv" "pause" "pause" "seek 5" "seek -5")`,
`'(“mpv” “–quiet” “pause\n” “pause\n” “seek 5\n” “seek -5\n”)`

 ref from emacs-china.

**Comment by SpringHan** [comment]
OK, I'll add it later.

**Comment by SpringHan** [comment]
I think list all the commands in README may be not very readable, I'll add this issue's link in README, You can put the commands here.  
And thanks for your contributions.

**Comment by SpringHan** [comment]
Now, because of the bug of `mplayer`, the original player change to `mpv`.

**Comment by askDing** [comment]
[Netease-Cloud-Music]: The repeat mode is in the initialization state. when you start playing song, it’ll be set. [11 times]

repeat mode dont work
when I press `r` in buffer of  Netease Cloud muisc , it appear `[Netease-Cloud-Music]: The repeat mode is in the initialization state. when you start playing song, it’ll be set. [11 times]
` in the minibuffer

**Comment by SpringHan** [comment]
> [Netease-Cloud-Music]: The repeat mode is in the initialization state. when you start playing song, it’ll be set. [11 times]
> 
> repeat mode dont work
> when I press `r` in buffer of Netease Cloud muisc , it appear `[Netease-Cloud-Music]: The repeat mode is in the initialization state. when you start playing song, it’ll be set. [11 times] ` in the minibuffer

Yes, because you should play a song to initialize it first. Then you can change it as you like.
